### PR TITLE
[MNGSITE-518] Update documentation about adding a skin in site descriptor

### PR DESCRIPTION
**Pull request by Bukama**

### CODE REVIEW

**Comment by Bukama** [comment]
@michael-o Requesting your review, because you were the one commenting the JIRA issue.

**Review by michael-o — APPROVED** [review]

File: content/apt/guides/development/guide-plugin-documentation.apt.vm [hunk 1]
```diff
@@ -215,12 +215,23 @@ public class ExampleMojo
...
+
+ Below is the suggested site descriptor template (with maven-fluido-skin).
  
 +--------------+  
 <?xml version="1.0" encoding="UTF-8"?>
 <project>
+  <skin>
+    <groupId>org.apache.maven.skins</groupId>
+    <artifactId>maven-fluido-skin</artifactId>
+    <version>2.0.1</version>
```
Comment (slawekjaranowski) [inline content/apt/guides/development/guide-plugin-documentation.apt.vm:232]: we can use:
```
${context.get("version.maven-fluido-skin")}
```

so we will have a version defined in pom

Comment (Bukama) [reply content/apt/guides/development/guide-plugin-documentation.apt.vm]: Updated and tested locally

File: content/apt/skins/index.apt [hunk 2]
```diff
@@ -86,7 +89,7 @@ Available Skins
   <skin>
     <groupId>org.apache.maven.skins</groupId>
     <artifactId>maven-fluido-skin</artifactId>
-    <version>1.12.0</version>
+    <version>${context.get("version.maven-fluido-skin")}</version>
```
Comment (slawekjaranowski) [inline content/apt/skins/index.apt:92]: we need add a `vm` extension to file to allow process Velocity content

Comment (Bukama) [reply content/apt/skins/index.apt]: Hm. In `apt.vm`files it works, but in `.apt` it does not. Reverted it.

Comment (slawekjaranowski) [reply content/apt/skins/index.apt]: so we need a file extension

Comment (Bukama) [reply content/apt/skins/index.apt]: Think so. For this PR I reverted it to the fixed version string

**Review by slawekjaranowski — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #1113

**Pull request merged**